README.md

```diff
@@ -34,11 +44,18 @@ You can sign up for your own app id at [https://dashboard.branch.io](https://das
 
 #### Unity Scene and Branch Parameters
 
-To allow Branch to configure itself, you must add a BranchPrefab asset to your first scene. Simply drag into your first scene, and then specify your `APP_KEY`, `PATH_PREFIX` and `APP_URI` in the properties.
+To allow Branch to configure itself, you must add a BranchPrefab asset to your first scene. Simply drag into your first scene, and then specify your `APP_KEY`, `APP_PREFIX` and `PATH_PREFIX`, `CUSTOM_DOMAIN`, `APP_LINKS` in the properties.
 
-* `branchKey`: This is your Branch key from the dashboard
-* `androidPathPrefix`: This is your Branch android path prefux [Read more](https://github.com/BranchMetrics/Android-Deferred-Deep-Linking-SDK/blob/master/README.md#leverage-android-app-links-for-deep-linking)
-* `branchUri`: This is the URI scheme you would like to use to open the app. This must be the same value as you entered in [the Branch link settings](https://dashboard.branch.io/#/settings/link) as well. Please do *not* include the `://` characters.
+* `Simulate Fresh Installs`: This is flag that enable or desable debud mode (in debug mode e your app will simulate fresh install each time, just for testing)
+* `Test Mode` : Switch set of parameters, if "Test mode" is enabled then app will use "test" set of parameters, else app will use "live" set of parameters
+* `APP_KEY`: This is your Branch key from the dashboard
+* `APP_URI`: <b>Keeped for supporting old realisation of link scheme.</b> This is the URI scheme you would like to use to open the app. This must be the same value as you entered in [the Branch link settings](https://dashboard.branch.io/#/settings/link) as well. Please do *not* include the `://` characters.
+* `PATH_PREFIX`: <b>Keeped for supporting old realisation of link scheme.</b> This is your Branch android path prefux [Read more](https://github.com/BranchMetrics/Android-Deferred-Deep-Linking-SDK/blob/master/README.md#leverage-android-app-links-for-deep-linking)
+* `CUSTOM_DOMAIN` : <b>Keeped for supporting old realisation of link scheme.</b> This is your custom domain.
+* `APP_LINKS` : This is your app links.
```

Comment:
@aaustin Alright i tried it and somehow the google ad id is only sent to the branch dashboard when using the "send "buy" event" button or the "send complex event" button.

![unbenannt1](https://cloud.githubusercontent.com/assets/3418387/18417394/c7fbb204-782f-11e6-95cd-f08e0c3b8ef6.PNG)

![unbenannt](https://cloud.githubusercontent.com/assets/3418387/18417389/5be9a198-782f-11e6-9a8a-f3104fc680bf.PNG)

I think it only sends the google ad id when using Branch.userCompletedAction("actionName"), because in actions like the normal "open" or even "install" events there is no google ad id sent over to the branch dashboard.

![unbenannt2](https://cloud.githubusercontent.com/assets/3418387/18417398/d53dd3de-782f-11e6-8237-17369a7cec09.PNG)

![unbenannt3](https://cloud.githubusercontent.com/assets/3418387/18417400/db78d276-782f-11e6-8ca1-1f95b535f846.PNG)

And if i understood it correctly this goodle ad id is needed for branch to connect to the facebook app (with the facebook app id and secret added on the dashboard) to send facebook these events.
That would imply that at least these "buy" events should be sent over to facebook. But, though my ad id is listed now under that buy event on the dashboard, i cant see the same event on the facebook side.